config/routes.rb

```diff
@@ -4,7 +4,9 @@
     sessions: "users/sessions"
   }
   resources :guilds, except: %i[destroy index] do
-    resources :games, only: %i[new create]
+    resources :games, only: %i[new create] do
+      resources :event_member, only: %i[new create destroy]
```

Comment:
I don't think we need to nest the event members

Reply:
changes done